geometry/clipping.cpp

```diff
@@ -86,7 +86,7 @@ void ClipTriangleByRect(m2::RectD const & rect, m2::PointD const & p1, m2::Point
     return;
   }
 
-  double constexpr kEps = 1e-8;
+  static constexpr double kEps = 1e-8;
```

Comment:
Why it is better?

Reply:
This variable should be static because it's used without explicit capture in lambda below.